sdk/storage/Azure.Storage.Blobs/src/Sas/BlobAccountSasPermissions.cs

```diff
@@ -92,20 +105,20 @@ public static BlobAccountSasPermissions Parse(string s)
         /// <returns>True if they're equal, false otherwise.</returns>
         [EditorBrowsable(EditorBrowsableState.Never)]
         public override bool Equals(object obj) =>
-            obj is BlobAccountSasPermissions other && this.Equals(other);
+            obj is BlobAccountSasPermissions other && Equals(other);
 
         /// <summary>
         /// Get a hash code for the BlobAccountSasPermissions.
         /// </summary>
         /// <returns>Hash code for the BlobAccountSasPermissions.</returns>
         [EditorBrowsable(EditorBrowsableState.Never)]
         public override int GetHashCode() =>
-            (this.Read   ? 0b000001 : 0) +
-            (this.Add    ? 0b000010 : 0) +
-            (this.Create ? 0b000100 : 0) +
-            (this.Write  ? 0b001000 : 0) +
-            (this.Delete ? 0b010000 : 0) +
-            (this.List   ? 0b100000 : 0);
+            (Read ? 0b000001 : 0) +
+            (Add ? 0b000010 : 0) +
```

Comment:
Does it complain about this whitespace?  Or were we being aggressive?

Reply:
Its due to this rule `csharp_space_around_declaration_statements = do_not_ignore`